### DISCUSSION TITLE
What attracts tourists to Wieliczka, PL every year?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#624] [Galaxy Trivia] [answered]
**Bonus:** How old is this place?

**Comment by peterbaccaro** [comment, marked as answer]
Wieliczka (located in southern Poland) is a popular tourist destination known for its **historic salt mine**, which has been in operation for over **700 years**. The Wieliczka Salt Mine is a UNESCO World Heritage site and one of the oldest and largest salt mines in the world.

**Comment by nsafarova** [comment]
Wieliczka, a town in southern Poland, is known for its historic salt mine which has been in operation since the 13th century and is a UNESCO World Heritage Site, which means it is more than 800 years old. Tourists are attracted to the mine's underground tunnels and chambers which feature sculptures, chapels, and an underground lake.
